workshops/models.py

```diff
@@ -422,7 +422,8 @@ def is_admin(user):
         return False
     else:
         return (user.is_superuser or
-                user.groups.filter(name='administrators').exists())
+                user.groups.filter(name='administrators').exists() or
+                user.groups.filter(name='steering committee').exists())
```

Comment:
Make this a one query.

Remember that `is_admin` is run almost on every page, so if you increase number of queries here, AMY's performance may decrease.